tests/Unit/Helpers/DataStructures/DataBox/TestHelpers.hpp

```diff
@@ -64,15 +74,23 @@ template <typename Tag>
 void test_prefix_tag(const std::string& expected_name) {
   static_assert(std::is_base_of_v<::db::PrefixTag, Tag>,
                 "A prefix tag must be derived from db::PrefixTag");
+  static_assert(::db::is_simple_tag_v<Tag>,
+                "A prefix tag should also be a simple tag");
+  static_assert(detail::has_type_v<Tag>);
+  static_assert(detail::has_tag_v<Tag>);
   detail::check_tag_name<Tag>(expected_name);
 }
 
 template <typename Tag>
 void test_simple_tag(const std::string& expected_name) {
-  static_assert(std::is_base_of_v<::db::SimpleTag, Tag> and
-                    not std::is_base_of_v<::db::ComputeTag, Tag>,
-                "A simple tag must be derived from "
-                "db::SimpleTag, but not db::ComputeTag");
+  static_assert(::db::is_simple_tag_v<Tag>,
+                "A simple tag must be derived from db::SimpleTag, but not "
+                "db::ComputeTag");
+  static_assert(detail::has_type_v<Tag>);
+  if constexpr (detail::has_base_v<Tag>) {
```

Comment:
I didn't realize simple tags can/should have a `base` alias. Don't they just derive from the base tag? What do they need the alias for?

Reply:
they need it if they are also derived from a `db::BaseTag`; otherwise they should not have a base class

Reply:
If I remember correctly many simple tags that derive from a base tag don't have a `base` alias (yet), are you going to add that requirement? Also, what is it needed for?

Reply:
I should be able to test that; I can add it in the next DataBox PR.  The `base` alias will be needed for the lookup that occurs when a `get` or `argument_tags` specifies a `db::BaseTag`; currently the DataBox finds the first matching tag in its `tags_list` that is derived from the specified `db::BaseTag`. With `if constexpr` I hope to simplify the lookup, but this will need the `base` alias in the `db::SimpleTag`